docs/src/language_servers.md

```diff
@@ -559,8 +559,16 @@ For development using the Deno framework follow installation instructions on [LS
 
 ## Vue
 
+There are multiple options:
+
+### Vetur
+
 Follow installation instructions on [LSP-vue](https://github.com/sublimelsp/LSP-vue).
 
+### Volar
+
+Follow installation instructions on [LSP-volar](https://github.com/sublimelsp/LSP-volar).
```

Comment:
Maybe mention that Volar is only for Vue 3?

Reply:
Volar can be setup to work with Vue 2. (See the Volar readme for more details)

The only major difference that I have seen is that vetur supports monorepos, while volar supports the script setup syntax.

Reply:
OK. A bit too much work to try. :)